tools/packaging/promtail.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Promtail service
+After=network.target
+
+[Service]
+Type=simple
+User=promtail
+ExecStart=/usr/bin/promtail -config.file /etc/promtail/config.yml
+
```

Comment:
Wouldn't it be a neat idea if promtail would also be `Restart = on-failure`, just like in `loki.service`?